packages/jellyfish-transaction-builder/src/txn/txn_builder_governance.ts

```diff
@@ -12,7 +12,12 @@ export class TxnBuilderGovernance extends P2WPKHTxnBuilder {
    * @returns {Promise<TransactionSegWit>}
    */
   async createCfp (createCfp: CreateCfp, changeScript: Script): Promise<TransactionSegWit> {
-    const creationFee = this.network.name === 'regtest' ? new BigNumber('1') : new BigNumber('10')
+    if (createCfp.nCycles > 100) {
+      throw new TxnBuilderError(TxnBuilderErrorType.INVALID_CFP_CYCLE,
+        'CreateCfp cycles should be between 0 and 100'
+      )
+    }
+    const creationFee = BigNumber.maximum(10, createCfp.nAmount.multipliedBy(0.01))
```

Comment:
```suggestion
    // Creation fee must 1% of the requested amount
    const creationFee = BigNumber.maximum(10, createCfp.nAmount.multipliedBy(0.01))
```

I would clarify that and add more comments/documentation to the method. Since it's an expensive operation that is implied by not explicit.